jobs/citation/OrcidJob.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * @file jobs/citation/OrcidJob.php
+ *
+ * Copyright (c) 2025 Simon Fraser University
+ * Copyright (c) 2025 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class OrcidJob
+ *
+ * @ingroup jobs
+ *
+ * @brief Job for retrieving structured metadata for citations from external services.
+ */
+
+namespace PKP\jobs\citation;
+
+use APP\core\Application;
+use APP\facades\Repo;
+use PKP\citation\job\externalServices\orcid\Inbound;
+use PKP\job\exceptions\JobException;
+use PKP\jobs\BaseJob;
+
+class OrcidJob extends BaseJob
+{
+    protected int $citationId;
+
+    public function __construct(int $citationId)
+    {
+        parent::__construct();
+
+        $this->citationId = $citationId;
+    }
+
+    public function handle(): void
+    {
+        $citation = Repo::citation()->get($this->citationId);
+
+        if (!$citation) {
+            throw new JobException(JobException::INVALID_PAYLOAD);
+        }
+
+        if ($citation->getIsProcessed()) {
+            return;
+        }
+
+        $authors = $citation->getData('authors');
+        if (empty($authors)) {
+            return;
+        }
+
+        $publication = Repo::publication()->get($citation->getData('publicationId'));
+        $submission = Repo::submission()->get($publication->getData('submissionId'));
+        $context = Application::getContextDAO()->getById($submission->getData('contextId'));
+        $contactEmail = $context->getContactEmail();
```

Comment:
maybe to provide contactEmail to each job, when the job is dispatched -- so that we do not need to get all the objects here...